projects/kubernetes/bridge.yml

```diff
@@ -0,0 +1,6 @@
+onboot:
+  - name: bridge
+    image: alpine:3.6
+    command: ["/bin/sh", "-c", "set -ex; mkdir -p /var/lib/cni/etc/net.d; echo '{\"cniVersion\":\"0.3.1\",\"name\":\"default\",\"plugins\":[{\"type\":\"bridge\",\"bridge\":\"cni0\",\"isDefaultGateway\":true,\"ipMasq\":false,\"hairpinMode\":true,\"ipam\":{\"type\":\"host-local\",\"subnet\":\"10.1.0.0/16\",\"gateway\":\"10.1.0.1\"},\"dns\":{\"nameservers\":[\"10.1.0.1\"]}},{\"type\":\"portmap\",\"capabilities\":{\"portMappings\":true},\"snat\":true}]}' > /var/lib/cni/etc/net.d/10-default.conflist; echo '{\"cniVersion\":\"0.2.0\",\"type\":\"loopback\"}' > /var/lib/cni/etc/net.d/99-loopback.conf"]
```

Comment:
the `mkdir` could be added to the runtime config for kubelet? This does seem a bit ugly...

Reply:
I'm not super happy with it either. I'd been thinking of https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/ so it could be a yaml to apply.

I'd also been considering having a directory somewhere which was the template/skeleton of this directory which could be applied at the same time as we unpack the default CNI things in `kubelet.sh`.

I think I'd like to get this batch of changes (particularly the `cri-containerd` bump) in now though and rework more in the future.

Reply:
Yeah sure, its not too terrible and its a new option anyway.